documentation/modules/configuring/proc-dedicated-nodes.adoc

```diff
@@ -3,49 +3,82 @@
 // assembly-scheduling.adoc
 
 [id='proc-dedicated-nodes-{context}']
-= Configuring pod affinity for dedicated nodes
+= Configuring pod scheduling for dedicated nodes
 
 [role="_abstract"]
 You can dedicate a set of worker nodes exclusively to your Kafka brokers so that no other applications can compete with Kafka for resources on those nodes.
 
-To configure dedicated worker nodes for Kafka pods in a specific pool, set taints on the worker nodes, and configure `tolerations`` and `nodeAffinity` in your `KafkaNodePool` resource.
+To configure dedicated worker nodes for Kafka pods in a specific pool, combine the following:
+
+Taints:: Apply taints to worker nodes to prevent other pods from being scheduled on them.
+Tolerations:: Apply tolerations to Kafka pods to allow them to be scheduled on tainted nodes.
+Affinity:: Apply affinity to Kafka pods to schedule them on specifically labeled nodes.
+
+The simplest way to configure this is to set a cluster-wide policy in the `Kafka` resource.
+For more granular control, if needed, you can override the default policy for specific node pools using the `KafkaNodePool` resource.
+This procedure shows example configurations for both.
+
+IMPORTANT: A `nodeAffinity` rule defined in a `KafkaNodePool` resource overrides any rule in the `Kafka` resource. 
```

Comment:
This is correct, but the situation is a bit more complicated ...
* If there is anything in `spec.template` in KafkaNodePool, the corresponding things in .spec.kafka.template` in Kafka are ignored.
* So, not only if there is node affinity configured in the node pool the node affinity from the Kafka CR will be ignored
* Even for examle if there is a container variable defined in `.spec.template.kafkaContainer` in KafkaNodePool, the affinity form `.spec.kafka.template in Kafka CR will be ignored.

I think we should make this more clear here. Maybe have something in one place explaining it in detail and having a link there from all places like this.